docker-compose.yml

```diff
@@ -23,6 +23,9 @@ services:
       - ~/.gitconfig:/root/.gitconfig
     ports:
       - '0.0.0.0:8080:8080'
+      - '0.0.0.0:33284:33284'  # Spells "DEBUG" in phone-number
```

Comment:
What is this for?

Reply:
To get a Python debugger in a Docker process, the best way is to use a remote debugger, that talks over an open port. I guess "Spells DEBUG in phone-number" doesn't adequately explain that 😆